src/transforms/grok_parser.rs

```diff
@@ -48,7 +56,18 @@ impl Transform for GrokParser {
         if let Some(value) = value {
             if let Some(matches) = self.pattern.match_against(&value) {
                 for (name, value) in matches.iter() {
-                    event.insert_explicit(name.into(), value.into());
+                    let name: Atom = name.into();
+                    let conv = self.types.get(&name).unwrap_or(&Conversion::Bytes);
+                    match conv.convert(value.into()) {
+                        Ok(value) => event.insert_explicit(name, value),
+                        Err(err) => {
+                            debug!(
+                                message = "Could not convert types.",
```

Comment:
```suggestion
                                message = "Could not convert type.",
```

Reply:
I copied that from the other (regex, tokenizer) type coercion changes, all say "types".